src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, Dispatch, SetStateAction } from 'react'
+import * as FaIcons from 'react-icons/fa'
+import * as AiIcons from 'react-icons/ai'
+import styles from './navbar.module.scss'
+import { AppRoutes, Route, appRoutes } from '../../views/router/routes'
+import classNames from 'classnames'
+
+type NavbarProps = {
+	setActiveTab: Dispatch<SetStateAction<Route>>
+	appRoutes: AppRoutes
+}
+
+const Navbar: React.FC<NavbarProps> = ({ setActiveTab, appRoutes }) => {
+	const [isSidebarOpen] = useState(true)
+	const betterNavClass = classNames(styles['nav-menu'], {
+		[styles.active]: isSidebarOpen,
+	})
+	return (
+		<>
+			<nav className={betterNavClass}>
+				<ul className={styles['nav-menu-items']}>
+					{Object.values(appRoutes).map((route, index) => (
+						<li
+							key={index}
+							className={styles['navbar-toggle']}
+							onClick={() => setActiveTab(route)}
+						>
+							{route.label}
+						</li>
```

Comment:
This needs to be styled according to the 2nd AC:
- The button will contain a square image
  - Don't design the images at this stage. Place holders will suffice

Also, this should be a button. 

Either that, or
``` C#
						<li key={index}>
							<button
								className={styles['navbar-toggle']}
								onClick={() => setActiveTab(route)}
							>
								{route.label}
							</button>
						</li>
```

Reply:
Honestly, I'd also make the argument that we probably don't need the ul and li structure anymore. I'm not super versed in semantic HTML, but I think the using the nav element (with role=navigation) satisfies your screen reader requirements (if you care about that)

``` Typescript
<nav className={styles['nav-menu'] + styles['active']} role='navigation'>
	{Object.values(appRoutes).map((route, index) => (
		<button
			key={index}
			className={styles['navbar-toggle']}
			onClick={() => setActiveTab(route)}
		>
			{route.label}
		</button>
	))}
</nav>
```